mkdocs.yml

```diff
@@ -10,11 +10,10 @@ nav:
   - Roadmap: roadmap.md
   - API reference: api/" target="_blank
   - We Made This (Community): we_made_this.md
-  - Core utilities:
+  - Features:
```

Comment:
Is it intentional to combine all features under one section?

Reply:
Yes, in the past weeks I had a talk with Heitor and saw that they are doing the same in their docs. The PR also moves the sub-menu navbar of each section on the right side. For a visual example check our currently deployed docs vs the Python ones.